src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -191,6 +179,7 @@ const NewSearchBarInput: FC<NewSearchBarInputProps> = ({ relay, viewer }) => {
 
   return (
     <AutocompleteInput
+      id="SearchBarInput"
```

Comment:
**question:** Is this `id` actually needed? (Or was it meant to be a test id? Since this UI element is present globally, could potentially cause an id collision.)

Reply:
Thanks, not needed for sure. I even removed it [last week](https://github.com/artsy/force/pull/12576/commits/344592a45ee9c6f66f6d4776b23b025fd97acd6c) but seems like I forgot to push changes